terraform/environments/cooker/macie.tf

```diff
@@ -0,0 +1,24 @@
+
+# Create macie account
+
+resource "aws_macie2_account" "macieaccess" { 
+   finding_publishing_frequency = "ONE_HOUR"
+   status                       = "ENABLED"
+ }
+
+# Now create a job
+
+resource "aws_macie2_classification_job" "test" {
+  job_type = "ONE_TIME"
+  name     = "JOBNAME"
+  s3_job_definition {
+    bucket_definitions {
+      account_id = "default_migration_source_account_id"
```

Comment:
Setting `account_id = "default_migration_source_account_id"` won't work. It needs an actual account ID. You should take a look at how the account ID can be retrieved programatically in other examples in this repository.

Reply:
I think this may be why it failed. I will look at the repo to see if there are other alternatives.

Reply:
Due to fiddling around with git this morning I lost the code changes I'd made.
This has now been amended. I have changed it to use a different version.

Reply:
An alternative (data.aws_ssm_parameter.modernisation_platform_account_id.value) was put in place.

Reply:
A corrected version was added.